src/Trace/TraceSpan.php

```diff
@@ -163,16 +156,6 @@ public function info()
     }
 
     /**
-     * Returns the info array for serialization.
-     *
-     * @return array
-     */
-    public function jsonSerialize()
```

Comment:
These objects are no longer meant to be directly sent to a json encoder. The reporter will want to format them correctly.